src/index.html

```diff
@@ -1,16 +1,64 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="big"
+    action=""
+    method="post"
+    data-qa="big">
+
+      <div class="field-big hover">
+
+        <input
+        class="field__button-big"
+        type="submit"
+        value=""
+        >
+        <input
+        class="field__input-big focus"
+        type="text"
+        name="default"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
+      </div>
+
+    </form>
+
+    <form
+    class="small"
+    action=""
+    method="post"
+    data-qa="small"
+    >
```

Comment:
```suggestion
    <form
      class="small"
      action=""
      method="post"
      data-qa="small"
    >
```
У атрибутов должен быть отступ 2 символа

Reply:
Исправил